introduction.md

```diff
@@ -0,0 +1,82 @@
+This repository contains a list of dojo rules for the Deadly Vipers dojo
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.